joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

joint1_position_controller:
  type: effort_controllers/JointPositionController
  joint: arm_joint_01
  pid: {p: 10.0, i: 0.01, d: 0.1}

joint2_position_controller:
  type: effort_controllers/JointPositionController
  joint: arm_joint_02
  pid: {p: 5.0, i: 0.01, d: 0.1}

joint3_position_controller:
  type: effort_controllers/JointPositionController
  joint: arm_joint_03
  pid: {p: 1.0, i: 0.01, d: 0.1}